src/components/checkbox/checkbox.template.tsx

```diff
@@ -38,6 +40,7 @@ export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = (
                     checked={props.checked}
                     disabled={props.disabled}
                     required={props.required}
+                    readonly={props.readonly}
```

Comment:
```suggestion
                    disabled={props.disabled || props.readonly}
                    required={props.required}
```

Reply:
I think this is better to do. We don't need to make the `input` readonly. I don't think it makes sense. But we can make it disabled, in case readonly = true

Reply:
Yes, I don't think `input type="checkbox"` understands the attribute `readonly`, so `disabled` should be used. The difference is only visual (and perhaps if there's any difference to what aria attributes that should be used (I'm not sure)).